sdk/core/Azure.Core/src/Shared/OperationInternal.cs

```diff
@@ -228,7 +228,7 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
 
         protected virtual async ValueTask<Response> UpdateStateAsync(bool async, CancellationToken cancellationToken)
         {
-            OperationState state = await _operation.UpdateStateAsync(async, cancellationToken).ConfigureAwait(false);
+            OperationState state = await _operation!.UpdateStateAsync(async, cancellationToken).ConfigureAwait(false);
```

Comment:
Can we guarantee that _operation is not null here?

Reply:
Fixed, now `_operation` isn't nullable.